Gemfile

```diff
@@ -9,7 +9,7 @@ gem 'aws-sdk-core'
 gem 'gchartrb', require: 'google_chart'
 gem 'gravtastic'
 gem 'high_voltage'
-gem 'highline'
+gem 'highline', '~> 1.7'
```

Comment:
When I first tried to install brakeman in the Gemfile, it failed because brakeman wanted highline 1.6.x. So, I promoted the 1.7 version constraint of highline from the Gemfile.lock to the Gemfile.

I'm not familiar with how the rg tracks and updates dependent gems.

If your gut it to remove it, I will, no problem.